hack/run_locally.sh

```diff
@@ -17,22 +17,25 @@ SECRET_LOCATION=$ROOT/tmp-secret-location/
 mkdir -p $SECRET_LOCATION
 
 platformtype=$(oc get infrastructures.config.openshift.io cluster  -o jsonpath='{.status.platform}')
+if [ -z "${CONTROLLER_NAMESPACE}" ]; then
```

Comment:
NIT: you can use a single liner
```
export CONTROLLER_NAMESPACE="${CONTROLLER_NAMESPACE:-openshift-cloud-network-config-controller}"
```

Reply:
Changed to:
~~~
   19 platformtype=$(oc get infrastructures.config.openshift.io cluster  -o jsonpath='{.status.platform}')
   20 export CONTROLLER_NAMESPACE="${CONTROLLER_NAMESPACE:-openshift-cloud-network-config-controller}"
   21 export CONTROLLER_NAME="${CONTROLLER_NAME:-tmp-local-controller}"
~~~